lib/workers/global/config/parse/cli.spec.ts

```diff
@@ -142,5 +142,25 @@ describe('workers/global/config/parse/cli', () => {
         Error("Invalid JSON value: 'Hello_World'")
       );
     });
+    it('dryRun boolean true', () => {
+      argv.push('--dry-run=true');
```

Comment:
please use a local copy of `argv` otherwise a failing tests will cause side effects on other following tests. or is `argv` reset before/ after each test? then the last line is obsolete.

Reply:
actually, I used the other tests in `cli.spec.ts` as examples. 
how should I use the local argv? or reset before/after each test?

Reply:
argv is already reset in before each. so remove your reset code in tests.